packages/client/src/me/sendInvite.ts

```diff
@@ -5,26 +5,32 @@ import {
 } from "@raha/api-shared/dist/routes/me/definitions";
 
 import { callApi } from "../callApi";
+import { VideoReference } from "@raha/api-shared/dist/models/VideoReference";
 
 /**
  * API call that sends an invite to join Raha to the provided email address.
- * @param inviteEmail Email address of invited new member
- * @param videoToken Token identifying invite video taken, if available
  */
-export function sendInvite(
-  apiBase: string,
-  authToken: string,
-  inviteEmail: string,
-  videoToken: string,
-  isJointVideo: boolean
-) {
+export function sendInvite(args: {
```

Comment:
What's the `args: { }` change for? Don't see anything different in the call-sites

Reply:
Just would rather be using named arguments than positional ones, less likely to mess them up, less brittle to stuff like mistaken orderings, better automatic code suggestions

Reply:
but i can save that for a separate pr